db/migrate/20230712044015_create_posts.rb

```diff
@@ -0,0 +1,14 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :comments_counter
+      t.integer :likes_counter
+      t.integer :author_id
+
+      t.timestamps
+    end
+    add_index :posts, :author_id
+  end
+end
```

Comment:
- [ ] Great work on creating the post table so far 👍 but i guess you should have added the author_id as a foreign because in the way you have done it not creating any foreign_key of author_id in the posts table so kindly make your migrations as you can see below to fix this issue ⬇️

```ruby
class CreatePosts < ActiveRecord::Migration[7.0]
  def change
    create_table :posts do |t|
      t.string :title
      t.text :text
      t.integer :comments_counter
      t.integer :likes_counter
      t.belongs_to :author, null: false, foreign_key: { to_table: :users }

      t.timestamps
    end
  end
end

```